group_vars/nfss.yml.sample

```diff
@@ -85,9 +85,8 @@ dummy:
 #nfs_containerized_deployment_with_kv: false
 #kv_type: etcd
 #kv_endpoint: 127.0.0.1
-#ceph_nfs_docker_username: ceph
-#ceph_nfs_docker_imagename: ganesha
-#ceph_nfs_docker_image_tag: latest
+#ceph_docker_image: "ceph/ganesha"
```

Comment:
Is `ceph/ganesha` a thing? I don't see it @ https://hub.docker.com/r/ceph/ . It looks like the ganesha packages are installed in the "ceph/daemon" image. (Separate PR to change this?)